README.md

```diff
@@ -298,8 +300,8 @@ This class needs also to be added to the `DATA_SOURCE_CLASSES` environment varia
 ### Properties
 
 * `info`: Descriptive text for the data source (can also be a multilingual dict)
-* `timeout`: Time you want to cache the data
-* `default`: The default value to be returned if execution of `get_data()` fails
```

Comment:
Why is this removed? As far as I can see, `timeout` is still supported?

Reply:
`timeout` is now set in the decorator and not as a property of the DataSource class anymore.

I did not mention the timeout arg in the README because I think in the context of a caching decorator, the word "timeout" pretty much says it all.

Reply:
Alright :+1: